crates/apub/src/protocol/objects/page.rs

```diff
@@ -111,6 +113,13 @@ impl Attachment {
       Attachment::Document(d) => d.url,
     }
   }
+
+  pub(crate) fn alt_text(self) -> Option<String> {
+    match self {
+      Attachment::Image(i) => i.name,
+      _ => None,
```

Comment:
We also need to read `Attachment::Document(i) => i.name,` for some reason thats how Mastodon federates images.